pdns/common_startup.cc

```diff
@@ -496,9 +496,7 @@ static void triggerLoadOfLibraries()
 
 void mainthread()
 {
-   Utility::srandom(time(0) ^ getpid());
```

Comment:
It looks like `random()` is still used in `waitForMultiData()`, `waitFor2Data()` (this one seems unused, though) and `Lua` records, so perhaps we still need to seed it properly?

Reply:
Yes, looking into this problem as we speak. e.g. pdns_notify uses random() for it's DNS id and always sends the same one.